app/lib/irs1040_pdf.rb

```diff
@@ -1,92 +1,87 @@
+      PhoneNumber: PhoneParser.formatted_phone_number(@xml_document.at("PhoneNum")&.text),
+      EmailAddress: @xml_document.at("EmailAddressTxt")&.text
     }
-    answers.merge!(spouse_info) if @tax_return.filing_jointly?
-    answers.merge!(dependents_info) if @qualifying_dependents.count.nonzero?
+    answers.merge!(spouse_info) if @xml_document.at("IndividualReturnFilingStatusCd")&.text.to_i == TaxReturn.filing_statuses[:married_filing_jointly]
+    dependent_nodes = @xml_document.search("DependentDetail")
+    answers.merge!(dependents_info(dependent_nodes)) if dependent_nodes.any?
     answers
   end
 
-  def bank_account_info
-    return {} unless @intake.bank_account.present?
-
-    {
-        RoutingTransitNum35b: pdf_mask(@intake.bank_account.routing_number, 4),
-        DepositorAccountNum35d: pdf_mask(@intake.bank_account.account_number, 4),
-        BankAccountTypeCd: @intake.bank_account.account_type.titleize
-    }
-  end
+  private
 
   def spouse_info
     {
-        Spouse65OrOlderInd: bool_checkbox(@tax_return.spouse_age_65_or_older?),
-        SpouseFirstNm: @intake.spouse_first_name,
-        SpouseLastNm: @intake.spouse_last_name,
-        SpouseSSN: @intake.spouse_ssn,
-        SpouseSignature: @intake.spouse_full_name,
-        SpouseSignatureDate: @intake.spouse_signature_pin_at.strftime("%m/%d/%y"),
-        SpouseIPPIN: @intake.spouse_ip_pin,
-        SpouseBlindInd: bool_checkbox(@intake.spouse_was_blind_yes?),
+      Spouse65OrOlderInd: bool_from_xml(@xml_document.at("Spouse65OrOlderInd")) ? "1" : "Off",
+      SpouseFirstNm: @spouse_first_name,
+      SpouseLastNm: @spouse_last_name,
+      SpouseSSN: @xml_document.at("SpouseSSN")&.text,
+      SpouseIPPIN: @xml_document.at("SpouseIdentityProtectionPIN")&.text,
+      SpouseBlindInd: bool_from_xml(@xml_document.at("SpouseBlindInd")) ? "1" : "Off",
     }
   end
 
   # TODO: The tax form only allows for 4 dependents. In the case where we have more than 4 dependents, we
   # really ought to attach a second page with dependent information.
-  def dependents_info
+  def dependents_info(dependent_nodes)
     answers = {}
-    @qualifying_dependents.first(4).each_with_index do |dependent, index|
-      answers["DependentLegalNm[#{index}]"] = dependent.full_name
-      answers["DependentRelationship[#{index}]"] = dependent.irs_relationship_enum
-      answers["DependentSSN[#{index}]"] = dependent.ssn
-      answers["DependentCTCInd[#{index}]"] = dependent.qualifying_ctc ? 1 : 0
+    dependent_nodes.first(4).each_with_index do |dependent, index|
+      answers["DependentLegalNm[#{index}]"] = [dependent.at("DependentFirstNm").text, dependent.at("DependentLastNm").text].join(" ")
+      answers["DependentRelationship[#{index}]"] = dependent.at("DependentRelationshipCd").text
+      answers["DependentSSN[#{index}]"] = dependent.at("DependentSSN").text
+      answers["DependentCTCInd[#{index}]"] = bool_from_xml(dependent.at("EligibleForChildTaxCreditInd")) ? 1 : 0
```

Comment:
its really weird that this is 1 or 0 and the others are 1 or off ... im now seeing why you dont have a helper for it. This seems like an inconsistency that could be fixed in the implementation of the PDF mapping if we thought it was necessary